test-src/typescript-consumer/index.ts

```diff
@@ -2,11 +2,14 @@
  * Note that as a consumer, you would import jsLogger and its
  * types using the following syntax:
  *
- *	import * as jsLogger from 'js-logger';
```

Comment:
@mutsys please can you confirm that the old `import * as jsLogger from 'js-logger'` syntax still works, otherwise this will be a breaking change for consumers.